File: test_resources/swift/struct_empty.swift
struct Empty: TechPawsBuffersModel {
    static func createBuffersDefault() -> Self {
        return Empty()
    }

    static func readFromBuffers(
        _ bytesReader: TechPawsBuffersBytesReader
    ) -> Self {
        return Empty()
    }

    static func skipInBuffers(
        _ bytesReader: TechPawsBuffersBytesReader,
        _ count: UInt64
    ) {
    }

    func writeToBuffers(
        _ bytesWriter: TechPawsBuffersBytesWriter
    ) {
    }
}

File: test_resources/kotlin/struct_empty.kt
class Empty() {
    companion object {
        fun createDefault(): Empty = Empty()

        fun readFromBuffers(reader: Long): Empty {
            return Empty()
        }

        fun skipInBuffers(reader: Long, count: Int) {
        }
    }

    fun writeToBuffers(writer: Long) {
    }
}
